### ISSUE TITLE
Join together outputs from multiple labeling jobs

### ISSUE THREAD

**Issue by koles289** [aws-samples/amazon-sagemaker-aws-greengrass-custom-object-detection-model#12]
Hi, I really liked this sample and it helped me an awfully lot. I am quite a newbaby to AWS so I would like to have one question regarding section of **Join together outputs from multiple labeling.** I have multiple labeling jobs and I would like to join labeled data from this jobs into one bigger dataset, but the data are stored in different s3 buckets. Do you think, it is possible, to join even these datasets? 
Because when I try it, I always get errors while training, so I am not sure, if it is even possible.
Thank you very much for you response, I would greatly appreciate it.

**Comment by Raunak-Singh-Inventor** [comment]
Why don't you just copy multiple outputs or manifests in different s3 buckets into one big manifest file in one s3 bucket and pass it to that step?

**Comment by koles289** [comment]
> Why don't you just copy multiple outputs or manifests in different s3 buckets into one big manifest file in one s3 bucket and pass it to that step?

Because when I tried to do it this simple way, I always got error during training. 

In fact, the answer to this question already is in the notebook, **_sagemaker_client.describe_labeling_job(LabelingJobName=label_job_id)['LabelingJobOutput']['OutputDatasetS3Uri']_**

this function is commented and I was blind enough not to see it. Using this, I was able to merge my datasets. 

But thank you very much for your answer, Raunak. 

I will close the issue now.
